### ISSUE TITLE
Plugin settings schema validation doesn't work.

### ISSUE THREAD

**Issue by ouvtk** [ExpressGateway/express-gateway#756]
### Issue
Plugin settings schema validation doesn't work.

### Steps to reproduce:
1. Install [Example Plugin](https://github.com/ExpressGateway/express-gateway-plugin-example).
2. Add simple schema to the plugin as described in [Plugin Development](https://www.express-gateway.io/docs/plugins/plugin-development/#manifestjs-file-overview-plugin-manifest) documentation:
```
  schema: {
    "$id": "example-schema",
    "should-be-missing": {
      "type": 'string',
    },
    "required": ["should-be-missing"]
  }
```
3. Run Express-Gateway. Notice that no error thrown and the plugin is still loaded.

### Expected 
Plugin loading should fail in case when settings doesn't pass plugin schema validation.

### Why so
Previously on schema registration (Commit [a042002](https://github.com/ExpressGateway/express-gateway/blob/a0420024eb0f51ad01b4e22b3fa10b1ea917cb57/lib/schemas/index.js#L39)) an error were thrown in case when settings can't pass validation.
In latest version it returns an object with validation result and a list of occurred errors. But this results aren't used anywhere, so the plugin loads regardless of the schema validation.

**Comment by ouvtk** [comment]
Proposed fix in #757
Not sure how to unit-test this behaviour, please advise.

**Comment by XVincentX** [comment]
Closed by #757
